docs/other.md

```diff
@@ -541,6 +541,76 @@ type 为 all 时，category 参数不支持 cost 和 free
 
 <Route author="TonyRL" example="/cdzjryb/zw/projectList" path="/cdzjryb/zw/projectList" radar="1"/>
 
+## 代理转换
+
+传递 URL 和转化规则，将 HTML/JSON 转换为 RSS
+
+### HTML
+
+在 `routeParams` 参数中以 query string 格式指定选项，可以控制提取数据
+
+| 键              | 含义                                                   | 接受的值 | 默认值                  |
+| --------------- | ------------------------------------------------------ | -------- | ----------------------- |
+| `title`         | 指定 RSS 的标题                                        | `string` | 从当前网页中取`<title>` |
+| `item`          | 通过 CSS 选择器查找 HTML 元素作为`item`元素            | `string` | html                    |
+| `itemTitle`     | 在`item`中通过 CSS 选择器查找 HTML 元素作为`title`元素 | `string` | `item`元素              |
+| `itemTitleAttr` | 获取`title`元素属性作为标题                            | `string` | 元素 text               |
+| `itemLink`      | 在`item`中通过 CSS 选择器查找 HTML 元素作为`link`元素  | `string` | `item`元素              |
+| `itemLinkAttr`  | 获取`link`元素属性作为链接                             | `string` | `href`                  |
+| `itemDesc`      | 在`item`中通过 CSS 选择器查找 HTML 元素作为`desc`元素  | `string` | `item`元素              |
+| `itemDescAttr`  | 获取`desc`元素属性作为描述                             | `string` | 元素 html               |
+
+<Route author="ttttmr" example="/proxy/html/https%3A%2F%2Fwechat2rss.xlab.app%2Fposts%2Flist%2F/item=div[class='post-content']%20p%20a" path="/html/:url/:routeParams" :paramsDesc="[' URL地址，需要经过URL编码', '转换规则']">
```

Comment:
```suggestion
<Route author="ttttmr" example="/proxy/html/https%3A%2F%2Fwechat2rss.xlab.app%2Fposts%2Flist%2F/item=div[class='post-content']%20p%20a" path="/html/:url/:routeParams" :paramsDesc="['URL地址，需要经过URL编码', '转换规则']">
```

The value of `routeParams` should be `encodeURIComponent`ed in the exmaple properly.
What you did is only `encodeURI`.